File: app/src/main/java/com/example/javakotlin/javakotlinoverview/java/Abstract.java
package com.example.javakotlin.javakotlinoverview.java;

abstract class Abstract {
    public void myMethod(){
        System.out.println("Hello");
    }
    abstract public void anotherMethod();
}

class Demo extends Abstract {
    public void anotherMethod() {
        System.out.print("Abstract method");
    }
    public static void main(String args[]) {
        Demo d = new Demo();
        d.myMethod();
        d.anotherMethod();
    }
}

File: app/src/main/java/com/example/javakotlin/javakotlinoverview/kotlin/Abstract.kt
package com.example.javakotlin.javakotlinoverview.kotlin

abstract class Abstract {
    fun myMethod(){
        println("Hello")
    }

    abstract fun  anotherMethod();
}

class Demo : Abstract() {
    override fun anotherMethod() {
        println("Abstract method");
    }
}
fun main(args: Array<String>) {
    var demo : Abstract = Demo()
    demo.myMethod()
    demo.anotherMethod()
}